topologies/mainnet.nix

```diff
@@ -9,22 +9,22 @@ let
 
   regions = {
     a = { name = "eu-central-1";   # Europe (Frankfurt);
-      minRelays = 35;
+      minRelays = 52;
```

Comment:
Thinking to leave `minRelays` as is for now, and instead increase `-N` to match number of cores available per instance.
Adding a commit implementing this.